test/integration/scheduler/queue_test.go

```diff
@@ -542,13 +542,13 @@ func TestCoreResourceEnqueue(t *testing.T) {
 		},
 		{
 			name:         "Pod rejected by the NodeUnschedulable plugin isn't requeued when another unschedulable node is created",
```

Comment:
You should just add 

```go
// This test case is valid only when QHint is enabled 
// because QHint filters out a node creation made in triggerFn.
enableSchedulingQueueHint: []bool{true},
```

Reply:
Thank you for suggestion.
The test case passed successfully.